common/http/src/main/java/io/helidon/common/http/ByteBufferDataChunk.java

```diff
@@ -25,11 +27,21 @@
  */
 final class ByteBufferDataChunk implements DataChunk {
 
+    private static final VarHandle IS_RELEASED;
+
+    static {
+        try {
+            IS_RELEASED = MethodHandles.lookup().findVarHandle(ByteBufferDataChunk.class, "isReleased", boolean.class);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            throw new RuntimeException(e);
```

Comment:
(Or `throw (Error)new ExceptionInInitializerError(e.getMessage()).initCause(e);`. Not a big deal either way.)